xen-bugtool

```diff
@@ -455,6 +458,10 @@ def no_unicode(x):
         return x.encode('utf-8')
     return x
 
+def log_output(x):
```

Comment:
Should `log_output` use `output`? Maybe calling it `output_and_log`? Otherwise we see calls to both with the same argument,

Reply:
Good idea!

Reply:
Had to do a bit of magic to fix the integration test, there's some invalid character that can't be output in octal format